v4/source/status.yaml

```diff
@@ -60,14 +60,22 @@
           $ref: '#/responses/Unauthorized'
 
   /users/status/ids:
-    get:
+    post:
       tags:
         - status
       summary: Get user statuses by id
       description: |
         Get a list of user statuses by id from the server.
         ##### Permissions
         Must be authenticated.
+      parameters:
```

Comment:
Hello @thePanz thanks for this update
just a small nit

can you change the following parameters to fix the build issue and also the fix the definition

 - `name` should be `post`
 - in the `schema` the items should be:
  ```
  items:
    type: string
  ```

the whole `parameters` section:
```
parameters:
    - name: post
       in: body
       description: List of user ids to fetch
       required: true
       schema:
           type: array
           items:
             type: string
```

Reply:
Updated the MR with your comments :)